en/sensor/sfxx_lidar.md

```diff
@@ -1,9 +1,11 @@
-# LightWare SF1XX Lidar
+# LightWare SF1X/SF02/LW20 Lidar
```

Comment:
@bkueng I've also updated this (lightware) lidar - for your review. Main point of concern is that I assume we still need to set SENS_SF0X_CFG for the lidar so that the system knows which particular device the driver is connected to?

Reply:
Yes this is correct, we still need that parameter (unfortunately)